content/zh/blog/2023/egress-sni/index.md

```diff
@@ -62,7 +62,35 @@ keywords: [traffic-management,gateway,mesh,mtls,egress,remote]
 
 端到端的请求流程如下图所示：
 
-{{< image width="90%" link="./egress-sni-flow.png" alt="具有任意域名的出口 SNI 路由" title="有任意域名的出口 SNI 路由" caption="有任意域名的出口 SNI 路由" >}}
+{{< image width="90%" link="./egress-sni-flow.svg" alt="具有任意域名的出口 SNI 路由" title="有任意域名的出口 SNI 路由" caption="有任意域名的出口 SNI 路由" >}}
+
+此图展示了通过 SNI 作为路由转发器向 `en.wikipedia.org` 发起出口 HTTPS 请求。
+
+* 应用程序容器
+
+    Application originates HTTP/TLS connection towards the final destination. Puts destination’s hostname into the SNI header. This TLS session is not decrypted inside the mesh. Only SNI header is inspected (as it is in cleartext).
```

Comment:
All en text shall be removed

Reply:
Sorry for that, not save the last changes, done.